src/main/java/org/example/repository/CustomerRepository.java

```diff
@@ -2,24 +2,94 @@
 
 import org.example.entity.Customer;
 
-public class CustomerRepository implements CrudRepository<Customer>{
+
+import java.sql.*;
+
+
+public abstract class CustomerRepository implements CrudRepository<Customer> {
```

Comment:
Don't make it abstract. the fact that is's glowing red should tell you that you did something wrong. In this case you added additional methods to the interface and didn't implement them. Remove those methods and remove abstract keyword.